.azure-pipelines/cpu-unit-test.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Just curious: Why can this PR speedup CPU pipeline? Because we remove the docker build time?

Reply:
yes, it will use pre-built image and skip `apt-get`, etc.